deploy/overlays/stanford_staging/base_bigtable_info.yaml

```diff
@@ -0,0 +1,8 @@
+project: datcom-store
```

Comment:
Not sure if there is an immediate benefit of multiple base_bigtable_info.yaml? Mean, we will almost always need to update multiple (10+) such files during a release.  For now, all dc managed custom instance should always track DC base tables